src/toil/common.py

```diff
@@ -674,6 +646,12 @@ def createBatchSystem(config, jobStore=None, userScript=None):
 
         return batchSystemClass(**kwargs)
 
+    @classmethod
+    def clean(cls, jobStoreLocator):
+        from toil.jobStores.abstractJobStore import AbstractJobStore
```

Comment:
There's a circular import if I add it at the top since abstractJobStore.py imports from job.py and job.py imports from common.py